barcodes_generator_abstract/models/barcode_generate_mixin.py

```diff
@@ -32,8 +30,6 @@ class BarcodeGenerateMixin(models.AbstractModel):
 
     generate_type = fields.Selection(
         string="Generate Type",
-        selection=_GENERATE_TYPE,
-        readonly=True,
```

Comment:
I don't understand that move. could you explain why you remove that two lines ?

Reply:
On recent Odoo versions, releated fields are readonly by default, so no need to write it explicitely. And selection= is not needed on a related field, it's automatic.